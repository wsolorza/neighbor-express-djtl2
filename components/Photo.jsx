import Content from './Content';

const Photo = ({ block }) => (<Content block={block}>
  <img src={block.image} alt={block.alt} />
</Content>
)

export default Photo;