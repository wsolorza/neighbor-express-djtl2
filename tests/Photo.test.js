import { shallow } from "enzyme";
import Photo from "../components/Photo.jsx";

let wrapper;

const block = {
  title: 'Test title',
  body_markdown: 'Test body',
  image: 'https://cdn2-www.dogtime.com/assets/uploads/2011/03/puppy-development.jpg',
  alt: 'golden retriever puppy'
};

beforeEach(() => {
  wrapper = shallow( <Photo block={block} />);
});

describe("Photo block", () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});