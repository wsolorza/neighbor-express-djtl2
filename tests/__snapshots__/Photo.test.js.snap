// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo block renders correctly 1`] = `
<Content
  block={
    Object {
      "alt": "golden retriever puppy",
      "body_markdown": "Test body",
      "image": "https://cdn2-www.dogtime.com/assets/uploads/2011/03/puppy-development.jpg",
      "title": "Test title",
    }
  }
>
  <img
    alt="golden retriever puppy"
    src="https://cdn2-www.dogtime.com/assets/uploads/2011/03/puppy-development.jpg"
  />
</Content>
`;
